Spring-Boot/Spring-Boot-Problem/src/main/java/tw/waterball/judgegirl/springboot/problem/controllers/ProblemController.java

```diff
@@ -66,14 +65,14 @@
     private final UploadProvidedCodeUseCase uploadProvidedCodeUseCase;
     private final UploadTestcaseIOUseCase uploadTestcaseIOUseCase;
     private final TokenService tokenService;
-    
+
     @GetMapping("/tags")
     public List<String> getTags() {
         return getAllTagsUseCase.execute();
     }
 
     @GetMapping
-    public List<ProblemItem> getProblems(@RequestHeader(value = "Authorization", required = false) String authorization,
+    public List<ProblemView> getProblems(@RequestHeader(value = "Authorization", required = false) String authorization,
```

Comment:
Should keep returning List<ProblemItem> in pagination
Return List<ProblemView> only when `ids` is present

Reply:
return `ResponseEntity<?>`